### DISCUSSION TITLE
Nova 2 transcription of multichannel, times are off

### DISCUSSION THREAD

**Discussion by cheike569** [deepgram/community#1008] [General help]
Using `nova-2` or `nova-2-phonecall` model and setting `multichannel` to `true`, the transcription times are off.

First word result for `nova-2`: **Wrong**
```
 {
    word: 'hello',
    start: 2.8799999,
    end: 3.12,
    confidence: 0.7920532,
    punctuated_word: 'Hello?'
  },

```

First word result for `enhanced` (old model): **Correct**

```
{
    word: 'hello',
    start: 0.03998,
    end: 0.53998,
    confidence: 0.7055664,
    punctuated_word: 'Hello?'
  },

```

Unfortunately, Github doesn't allow for audio files to be attached.

I'm using the JS SDK like this

```
 const {result, error} = await deepgram.listen.prerecorded.transcribeFile(
        fs.readFileSync(file),
        {
            model: "enhanced",
            punctuate: true,
            multichannel: true,
        }
    );
```

**Comment by deepgram-community** [comment]
Hey there! It looks like you haven't connected your GitHub account to your Deepgram account. You can do this at https://community.deepgram.com - being verified through this process will allow our team to help you in a much more streamlined fashion.

**Comment by deepgram-community** [comment]
It looks like we're missing some important information to help debug your issue. Would you mind providing us with the following details in a reply?

- A request ID that triggered your error or issue.

**Comment by jkroll-deepgram** [comment]
Hi @cheike569, are you able to share the Deepgram request ID, and/or a link to the audio (such as in Google Drive), so we can take a closer look?
